gazebo_plugins/scripts/gazebo_model

```diff
@@ -15,32 +15,31 @@ from gazebo_plugins.msg import GazeboModel
 from geometry_msgs.msg import Pose, Point, Quaternion
 
 def usage():
-    print '''Commands:
-    load [param|file] [urdf|gazebo] <param/file name> <model name> <initial pose: x y z r p y> <namespace> - Load model
-    delete <model name>                                                        - Deletes the model named <model name>
-    '''
+    print('Commands:')
+    print('load [param|file] [urdf|gazebo] <param/file name> <model name> <initial pose: x y z r p y> <namespace> - Load model')
```

Comment:
nit, this changes the whitespace in the output message from `____load` to `load`. I think putting `()` around the `'''...'''` should be fine.

Reply:
switched to `('''...''')` in 8c1cad7